src/viewmodel/prototype/get.js

```diff
@@ -62,6 +63,12 @@ export default function Viewmodel$get ( keypath, options ) {
 		value = wrapped.get();
 	}
 
+	if ( keypath.isRoot && options.fullRootGet ) {
+		for ( key in this.mappings ) {
+			value[ key ] = this.mappings[ key ].getValue();
+		}
```

Comment:
Yes, good point. Does that mean it needs to happen prior to the wrapper so that this would work:

```js
var data = ractive.get();
data.foo = 'bar';
```

Or we expose as explicit option, or maybe it's own method and not guess about it:

```js
// how would an option work?
var data = ractive.get({ mappings: true });
var data = ractive.get( '', { mappings: true });

// what happens here?
var data = ractive.get( 'foo', { mappings: true });

// maybe just a dedicated method would be best: 
var data = ractive.getAll();
```

I'm leaning towards the last so we stay out of all the guess work. Thoughts?